src/Components/README.md

```diff
@@ -123,4 +136,4 @@ dotnet test --no-build
 
 ## More Information
 
-For more information, see the [ASP.NET Core README](../../README.md).
+For more information, see the [ASP.NET Core README](https://github.com/dotnet/aspnetcore/blob/main/README.md).
```

Comment:
I think this is going to cause problems as release branches get snapped.

Reply:
This is something we are already doing in most READMEs, and here I just want to prevent a dead link (or a bad one) when the folder moves.